refactor/src/Cage.py

```diff
@@ -474,7 +477,7 @@ def get_variance_infected_fish(self, n: int, k: int) -> float:
         probs = np.full(n, 1/n)
         bins = self.cfg.rng.multinomial(k, probs)
 
-        return np.var(bins) * k
+        return float(np.var(bins) * k)
```

Comment:
We need to choose to which datatype to stick for scalar values. This applies both here and in a number of places (eg. the parametrers of update_deltas, why some are int64 and other are int?) and mypy is extremely unhappy about this.

Reply:
I think this deserves separate issue - added #84